<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingRecordDialog</class>
 <widget class="QDialog" name="SettingRecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>動画設定</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>画像フォーマット</string>
   </property>
   <widget class="QRadioButton" name="radioButtonAVI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>AVI (.avi)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonMPEG1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;低速のPCでは動画生成が間に合わず、メモリーオーバーとして動画生成が中断されることがあります&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>MPEG1 (.mpeg)</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>生成する動画の解像度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>動画のフレームレート</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxFrameRate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>30.000000000000000</double>
   </property>
   <property name="value">
    <double>30.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>MPEG1では30が固定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>枚/秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1つの動画ファイルの最長動画時間の目安</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMaxMovieTime">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>227</red>
         <green>227</green>
         <blue>227</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>XXX時間YYY分ZZZ秒</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxDivideMovieFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>動画ファイル分割</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>長時間撮影の場合、ファイルを分けて生成します</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>設定</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>キャンセル</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>2番目以降のファイル名に-1 ,-2,...が付加されます</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCamMovieSize">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>幅：2592 × 高さ：1944</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>幅：2048 × 高さ：1536</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>幅：1920 × 高さ：1080</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>幅：1024 × 高さ：768</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>幅：640 × 高さ：480</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>幅：320 × 高さ：240</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
